### PR TITLE
Improved Posted Links & Subscribe button look like Youtube/Reddit style

### PR DESCRIPTION
**Pull request by defnax**
- Added a Combobox for Sort in Posted links easyer with new icons
- Messengerwindow changed the button icons & display on status button the status icon too.

### CODE REVIEW

**Comment by defnax** [comment]
* Improved Subscribe Button look and feel with stylsheet
* Removed icons from Subscribe button
* Subscribe button hided on posted too when subscribed

- Enabled back Subscribe Button on Channels
- Set stylesheet for the Channel subscribe button too.
- Fixed spacing on PostedListwidget

**Comment by felisucoibi** [comment]
thanks for your improveents

File: retroshare-gui/src/gui/MessengerWindow.cpp [hunk 1]
```diff
@@ -108,20 +108,22 @@ MessengerWindow::MessengerWindow(QWidget* parent, Qt::WindowFlags flags)
         }
         expandedGroups.clear();
 
-    ui.messagelineEdit->setMinimumWidth(20);
```
Comment (csoler) [inline retroshare-gui/src/gui/MessengerWindow.cpp:111]: Never use absolute size. Always use the font size times some factor!

**Comment by csoler** [comment]
Suggestions for improvements:
- remove the absolute size. It really looks weird on high DPI screens. Always use font size times some factor (search for QFontMetricsF in the code)
- in Channel post creation window, it would be much nicer to replace the buttons "add file" and "attach picture" by tool buttons with a proper icon. The "Add channel thumbnail" can be removed if you allow to change the thumbnail by clicking on it (that system exists already in other places of the UI)
- in channel creation window, you can use drag+drop of pictures. Now it just pasts the address of the image. It should paste the image itself.
-  text sizes are not consistent. *do not* fix font size manually in qtcreator. Let Qt find the correct size when it needs to display the text. It really looks bad here ;-) That is for the "subscribed" button in channels+forums+links, and the "create post" buttons. Also the "SORT" text is far too big as compared to the others. Same remark: let Qt find the proper sizes. I would actually remove the "SORT" text. It's not really useful. Just tooltip the combo box with "Select sorting"

screenshot from 2019-01-17 21-21-43

**Comment by defnax** [comment]
> * remove the absolute size. It really looks weird on high DPI screens. Always use font size times some factor (search for QFontMetricsF in the code)

i can remove it.

> * in Channel post creation window, it would be much nicer to replace the buttons "add file" and "attach picture" by tool buttons with a proper icon. The "Add channel thumbnail" can be removed if you allow to change the thumbnail by clicking on it (that system exists already in other places of the UI)

i has not much time to improve it more i have lots of other things todo you have always not time too.  ;)

> * in channel creation window, you can use drag+drop of pictures. Now it just pasts the address of the image. It should paste the image itself.
> * text sizes are not consistent. _do not_ fix font size manually in qtcreator. Let Qt find the correct size when it needs to display the text. It really looks bad here ;-) That is for the "subscribed" button in channels+forums+links, and the "create post" buttons. Also the "SORT" text is far too big as compared to the others. Same remark: let Qt find the proper sizes. I would actually remove the "SORT" text. It's not really useful. Just tooltip the combo box with "Select sorting"

i can remove the sort text.
but i doesnt added to qt ui fonts i added to the default stylesheet to see better the subscribe button like on YouTube/Reddit look similar.

I has not much time i did my best other people do nothing i started only to help little bit.
Im always waiting that some people imrove the ui and look and feel and nice to have things 
but it comes nothing.

**Comment by csoler** [comment]
yeah. I have the same problems then you ;-) This can wait. It's not a critical fix and even if it improves the look and feel of GUI, it can be merged later than 0.6.5 release when it's ready. Thx for the job anyway!

**Comment by felisucoibi** [comment]
thxs.

**Comment by defnax2k** [comment]
hi @csoler , soon two month where i created this pr, you get soon time? and after that you can fix this https://github.com/RetroShare/RetroShare/pull/1471

**Pull request merged**